server/types.py

```diff
@@ -27,3 +29,31 @@ class Map(NamedTuple):
     id: int
     name: str
     path: str
+    weight: int
+
+    def get_map(self) -> "Map":
+        return self
+
+
+class NeroxisGeneratedMap(NamedTuple):
+    id: str
+    version: str
+    spawns: int
+    map_size_pixels: int
+    weight: int
+
+    def get_map(self) -> Map:
+        """
+            Generate a map name based on the version and parameters. If invalid parameters are specified
+            hand back None
+        """
+        seed_bytes = random.getrandbits(64).to_bytes(8, 'little')
+        size_byte = int(self.map_size_pixels / 64).to_bytes(1, 'little')
+        spawn_byte = self.spawns.to_bytes(1, 'little')
+        option_bytes = spawn_byte + size_byte
+        seed_str = base64.b32encode(seed_bytes).decode("ascii").replace("=", "").lower()
+        option_str = base64.b32encode(spawn_byte + size_byte).decode("ascii").replace("=", "").lower()
```

Comment:
Already have `option_bytes` and b32encode is already guaranteed to return lowercase.
```suggestion
        option_str = base64.b32encode(option_bytes).decode("ascii").replace("=", "")
```

Reply:
good catch on the option bytes. Although when I tested it locally on my machine b32encode returned on uppercase. Is there a configuration somewhere else that forces it lower case?

Reply:
Hmm, I guess not. My bad.

Reply:
From what I can tell the default for b32encode and b32decode is uppercase